paywall/src/__tests__/utils/validators.test.js

```diff
@@ -666,6 +666,85 @@ describe('Form field validators', () => {
           ).toBe(false)
         })
       })
+
+      describe('persistentCheckout', () => {
+        it("should accept or 'true' as value", () => {
```

Comment:
yes, I am worried about someone passing 'false'. The config is a JSON object which could be "generated" from a form (see donations...) and in that case, the type could be messed up.

I think it is important that we stay liberal on the input side... rather than strict ;)